src/pages/service.tsx

```diff
@@ -195,7 +203,7 @@ const ServicePage = ({
       });
 
       if (!url) {
-        const subject = 'Here is a new service to track in Open Terms Archive';
+        const subject = 'Here is a new service to track in Open Terms Archive';
```

Comment:
Please do not modify whitespace. These non-breakable spaces are here [voluntarily](https://github.com/OpenTermsArchive/engine/wiki/Styleguide#common-to-all-languages).